webpack.config.js

```diff
@@ -149,7 +159,7 @@ module.exports = {
 			},
 
 			{
-				test: /\.scss(\.erb)?$/,
+				test: /\.s[ac]ss(\.erb)?$/i,
```

Comment:
I think we don't have .sass files, do we need to support those as well?

Reply:
I just copy pasted it from the documentation, but I think I can change it back to the original one.

Reply:
It's the recommendation, we might not need it but it doesn't hurt.